idb/postgres/internal/encoding/types.go

```diff
@@ -106,3 +106,29 @@ type specialAddresses struct {
 	FeeSinkOverride     crypto.Digest `codec:"FeeSink"`
 	RewardsPoolOverride crypto.Digest `codec:"RewardsPool"`
 }
+
+type baseOnlineAccountData struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	VoteID          crypto.OneTimeSignatureVerifier `codec:"A"`
+	SelectionID     crypto.VRFVerifier              `codec:"B"`
+	VoteFirstValid  basics.Round                    `codec:"C"`
+	VoteLastValid   basics.Round                    `codec:"D"`
+	VoteKeyDilution uint64                          `codec:"E"`
+}
+
+type baseAccountData struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	Status                     basics.Status `codec:"a"`
+	AuthAddr                   crypto.Digest `codec:"b"`
+	TotalAppSchemaNumUint      uint64        `codec:"c"`
+	TotalAppSchemaNumByteSlice uint64        `codec:"d"`
+	TotalExtraAppPages         uint32        `codec:"e"`
+	TotalAssetParams           uint64        `codec:"f"`
+	TotalAssets                uint64        `codec:"g"`
+	TotalAppParams             uint64        `codec:"h"`
+	TotalAppLocalStates        uint64        `codec:"i"`
```

Comment:
Not necessarily. It's good for space efficiency, and this is how it's going to be done in go-algorand. https://github.com/algorand/go-algorand/blob/fcd06e94f60d2141df0cd11ff6e3fb0518483cdc/ledger/accountdb.go#L1079

Reply:
Yes, for instance, it would show up in the postgres query log. In this PR that only applies to `auth-addr`.

Reply:
And yes, if someone had their own query to get any of those other renamed fields, those queries would break (that's what I was trying to get at in my first comment)

Reply:
I noticed that flipside crypto lets you query JSON keys inside the accountdata blob. names like onl, vote, voteFst, voteLst are being renamed here, more than auth-addr, right? @barnjamin

Reply:
Yes thats correct